src/components/Loading.tsx

```diff
@@ -14,28 +15,32 @@ const StyledLoading = styled.div`
   align-items: center;
 `;
 
-const rotate = keyframes`
-  0% {
-    transform: rotate(0deg);
-  }
-  100% {
-    transform: rotate(360deg);
-  }
-`;
-
-const Animation = styled.img`
+const Circle = styled.div`
+  width: 200px;
+  height: 200px;
   border-radius: 50%;
-  width: 100px;
-  height: 100px;
-  filter: contrast(1.02);
-  user-select: none;
-  animation: ${rotate} 2s linear 0s infinite;
+  overflow: hidden;
+
+  div {
+    transform: scale(1.3) translateY(20px);
+  }
 `;
 
 const Loading: React.FC = () => {
+  const defaultOptions = {
```

Comment:
This const should either be placed outside of the component or be wrapped in a useMemo to prevent it from re-rendering every time its parent changes.

Reply:
Thanks, moved outside 😄